src/CommonITILObject.php

```diff
@@ -483,12 +483,22 @@ protected function setPredefinedFields(ITILTemplate $tt, array &$options, array
         if (in_array($this->getType(), ['Change', 'Problem']) && $tickets_id) {
             $ticket->getFromDB($tickets_id);
 
-            $options['content']             = $ticket->fields['content'];
-            $options['name']                = $ticket->fields['name'];
-            $options['impact']              = $ticket->fields['impact'];
-            $options['urgency']             = $ticket->fields['urgency'];
-            $options['priority']            = $ticket->fields['priority'];
-            if (isset($options['tickets_id'])) {
+            $fields = [
```

Comment:
```suggestion
            // copy fields from original ticket, only when fields are not already set by the user (contained in _saved array)
            $fields = [
```

add a comment to clarify the feature, like seen together, it was hard without explanation to find how this works